### PR TITLE
Update webpack

### PR DESCRIPTION
**Pull request by igorline**

### CODE REVIEW

File: packages/kyodo-app/src/components/ContractData.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+    // Optionally convert to UTF8
+    if (this.props.toUtf8) {
+      displayData = drizzle.web3.utils.hexToUtf8(displayData);
+    }
+    // Optionally convert to Ascii
+    if (this.props.toAscii) {
+      displayData = drizzle.web3.utils.hexToAscii(displayData);
+    }
+    if (displayData instanceof Array) {
```
Comment (jsidorenko) [inline packages/kyodo-app/src/components/ContractData.js]: you could use new approach: `Array.isArray(displayData)`

File: packages/kyodo-contracts/test/TestKyodoDAO.test.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import truffleAssert from 'truffle-assertions';
-import assertRevert from 'openzeppelin-solidity/test/helpers/assertRevert';
+import { shouldFail } from 'openzeppelin-test-helpers';
```
Comment (jsidorenko) [inline packages/kyodo-contracts/test/TestKyodoDAO.test.js]: 👍 looks more descriptive now

Comment (igorline) [reply packages/kyodo-contracts/test/TestKyodoDAO.test.js]: https://github.com/OpenZeppelin/openzeppelin-test-helpers/ seems to be published very recently and has no issues with transpilation within truffle

File: packages/kyodo-app/src/components/ContractData.js (hunk 1 above)
Comment (igorline) [reply packages/kyodo-app/src/components/ContractData.js]: :+1: I'll keep it like that by now to keep consistency with prior is object check

**Pull request merged**